Samples/Samples/ViewModel/LauncherViewModel.cs

```diff
@@ -13,17 +12,33 @@ public class LauncherViewModel : BaseViewModel
 
         public ICommand CanLaunchCommand { get; }
 
+        public ICommand LaunchMailCommand { get; }
+
+        public ICommand LaunchBrowserCommand { get; }
+
         public LauncherViewModel()
         {
             LaunchCommand = new Command(OnLaunch);
+            LaunchMailCommand = new Command(OnLaunchMail);
+            LaunchBrowserCommand = new Command(OnLaunchBrowser);
             CanLaunchCommand = new Command(CanLaunch);
         }
 
+        async void OnLaunchBrowser()
+        {
+            await Launcher.OpenAsync("https://github.com/xamarin/Essentials");
```

Comment:
I don't think we want the sample to open the browser as that is what the Browser API is for and should guide developers down that route :) Even though technically it can be.

Reply:
You are correct, but I can not think of a uri example which works on every platform and is not covered by existing or by a PR API